packages/dd-trace/src/span_processor.js

```diff
@@ -54,6 +54,7 @@ class SpanProcessor {
     }
 
     if (this._killAll) {
+      // eslint-disable-next-line array-callback-return
```

Comment:
Why is it a `.map` then? Similar question for all other occurrences.

Reply:
Looks like the author intended to use a forEach?

Reply:
Then I'd say to change the code instead of adding an eslint rule.